cfn/service.yml

```diff
@@ -12,10 +12,10 @@ Parameters:
     Description: Select at least two private subnets in given VPC where Fargate run tasks
 Resources:
   # Load balancer security group
-  LoadBalancerSG:
```

Comment:
Don't change resource name, if name conflicts with existing resource, then change group name by adding something like project name, in this case "discovery" to the resource name. It applies to wherever you changed resource name